model/src/main/kotlin/utils/DefaultResolutionProvider.kt

```diff
@@ -17,15 +17,15 @@
  * License-Filename: LICENSE
  */
 
-package org.ossreviewtoolkit.reporter
+package org.ossreviewtoolkit.model.utils
 
 import org.ossreviewtoolkit.model.OrtIssue
 import org.ossreviewtoolkit.model.OrtResult
 import org.ossreviewtoolkit.model.RuleViolation
 import org.ossreviewtoolkit.model.config.Resolutions
 
 /**
- *
+ * A default provider of resolutions for [OrtIssue]s.
```

Comment:
@mnonnenmacher I'd appreciate if you can suggesting some more meaningful docs here that explains *how* issues are resolved by default.

Reply:
I want to get this merged, will eventually improve the wording later.

Reply:
This also provides resolutions for `[RuleViolation]s`. The only thing I would add here is how this class works, something like "resolutions can be added using the [add] function". How resolutions work is documented in the respective `RuleViolationResolution` and `IssueResolution` (which should maybe be renamed to `OrtIssueResolution`) classes.